Pipfile

```diff
@@ -5,8 +5,8 @@ name = "pypi"
 
 [packages]
 cookiecutter = "==2.4.0"
-urllib3 = "==2.1.0"
-django = "==3.2.25"
+urllib3 = "==2.2.2"
+django = "==4.2.16"
```

Comment:
Isn't the scaffold and our own app a Django version behind at 3.x? This version is codependent on the scaffold being in the same version

Reply:
The slack-app Django version is `4.2.16`. The scaffold version is at `3.2.25`. We probably need to also upgrade the scaffold Django version